fmriprep/workflows/base.py

```diff
@@ -353,10 +353,10 @@ def init_single_subject_wf(subject_id, task_id, name,
         raise Exception("No T1w images found for participant {}. "
                         "All workflows require T1w images.".format(subject_id))
 
-    if t2s_coreg and not layout.get_echos():
-        raise Exception("No multiecho BOLD images found for participant {}. "
-                        "T2* coregistration requires multiecho BOLD.").format(
-                            subject_id)
+    # if t2s_coreg and not layout.get_echos():
```

Comment:
Can we remove these commented out lines?

Reply:
I had an exception there, but I thought better of it. I still think something should be provided to the user, though, if they request `--t2s_coreg` with non-ME data. What do you think of a warning?

Reply:
A warning would be great!